packages/expect/src/jest-expect.ts

```diff
@@ -330,7 +330,7 @@ export const JestChaiExpect: ChaiPlugin = (chai, utils) => {
     if (Array.isArray(args[0]))
       args[0] = args[0].map(key => String(key).replace(/([.[\]])/g, '\\$1')).join('.')
 
-    const actual = this._obj
+    const actual = this._obj as any
```

Comment:
Not sure why but typescript was complaining this line on my IDE, so I added explicit `any`
![image](https://github.com/vitest-dev/vitest/assets/4232207/3af86804-49d8-4653-b576-da50e5773d7b)

Rollup dts is fine without this, so probably it's my local setup issue. If you see this unnecessary, then I can revert this, so please let me know.